File: c++/image_read.cpp
#include "opencv2/highgui/highgui.hpp"
#include <iostream>
using namespace cv;
using namespace std;
int main()
{
	Mat img = imread("1.jpg", CV_LOAD_IMAGE_UNCHANGED);
	if (img.empty())
	{
		cout << "Error : Image cannot be loaded..!!" << endl;
		system("pause");
		return -1;
	}
	imshow("Image", img);
	//wait infinite time for a keypress
	waitKey(0);			
	//destroy all window
	destroyAllWindows();
	return 0;
}

File: python/image_read.py
# import thư viện OpenCV
import cv2
  
# gán file ảnh 01.jpg vào biến img
img = cv2.imread("01.jpg")
  
# đặt tên cửa sổ hiển thị file ảnh 
# cv2.WINDOW_AUTOSIZE -> autosize window theo ảnh
# cv2.WINDOW_NORMAL -> fit ảnh trong window
cv2.namedWindow("Original RGB 1", cv2.WINDOW_AUTOSIZE)
cv2.namedWindow("Original RGB 2", cv2.WINDOW_NORMAL)
  
# hiển thị file ảnh img
cv2.imshow('Original RGB 1', img)
cv2.imshow('Original RGB 2', img)
  
# đợi người dùng nhấn phím bất kì
cv2.waitKey(0)
  
# xóa toàn bộ cửa sổ đã tạo ra
cv2.destroyAllWindows()